src/coprocessor/endpoint.rs

```diff
@@ -139,6 +139,8 @@ impl<E: Engine> Endpoint<E> {
                         store,
                         req_ctx.deadline,
                         batch_row_limit,
+                        is_streaming,
+                        true,
```

Comment:
I think we should make it configurable

Reply:
Yes, that's exactly the purpose of this flag. Before 3.0 release we should make it configurable and enables it by default (but disables for some critical users). It is easy to change it in the future. For now, let's just use this value because we want integration tests to test it.